ui/text.go

```diff
@@ -15,3 +15,23 @@ func Bold(text string) string {
 	color := Color(os.Stdout)
 	return color.Sprintf(color.Bold(text))
 }
+
+func RedText(payload string) aurora.Value {
+	color := Color(os.Stdout)
+	return color.Red(payload).Bold()
```

Comment:
Should all the colours be bold as well? IMO the `RedText` function should only change the color.